targets/TARGET_Cypress/TARGET_PSOC6/mbed_overrides.c

```diff
@@ -53,3 +56,19 @@ void mbed_sdk_init(void)
     __enable_irq();
 #endif
 }
+
+/*******************************************************************************
+* Function Name: mbed_main
+****************************************************************************//**
+*
+* Mbed's post-memory-initialization function.
+* Used here to initialize common parts of the Cypress libraries.
+*
+*******************************************************************************/
+void mbed_main(void)
```

Comment:
shouldn't this be `mbed_sdk_init` ? mbed_main is users code hook

Reply:
Removed custom `mbed_main` from mbed_overrides.c.

Cannot execute `cybsp_wifi_init` in `mbed_sdk_init` since WHD initialization flow requires RTOS to be up and running.